website/static/css/style.css

```diff
@@ -186,6 +186,16 @@ a {
         width: 100%;
         margin-bottom: 20px;
     }
+    /* Allow project nav to move the content on small screen */
```

Comment:
@abought Yes, you are correct! I should move that to project_base.css! Good catch, Thank you very much!